readme.md

```diff
@@ -344,7 +344,9 @@ A custom detector is a function that allows specifying custom detection mechanis
 
```

Comment:
This is out of sync now.

Reply:
It is out of sync, which I did not do in purpose, but syncing it with the `core.d.ts` does not make more readable.
They are not describing the exact same thing. `core.d.ts` is describing the custom detector type / interface, while the README is describing it having the header above as the only reference.